frontend/src/theme/spacing.scss

```diff
@@ -0,0 +1,5 @@
+.LinkSide,
+.MaxAndMinInputs,
+.NameContainer {
+  --el-border-radius-base: 10px;
```

Comment:
In these kind of files, variables should just be declared, and then used in other files (base yourself in colors.scss file)